app/controllers/api_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApiController < ApplicationController
-  http_basic_authenticate_with name: ENV['API_BASIC_AUTH_USERNAME'], password: ENV['API_BASIC_AUTH_PASSWORD']
+  # http_basic_authenticate_with name: ENV['API_BASIC_AUTH_USERNAME'], password: ENV['API_BASIC_AUTH_PASSWORD']
```

Comment:
Why is this being commented out - won't this break the HTTP BASIC auth (which we use)